pages/app/calendar.js

```diff
@@ -1,7 +1,69 @@
-// Calendar Widget
+// Calendar Widget //
+import Head from 'next/head';
+import { useState, useEffect } from 'react';
+import { styled } from '@mui/material/styles';
+import Paper from '@mui/material/Paper';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+  TodayButton,
+  AppointmentTooltip,
+} from '@devexpress/dx-react-scheduler-material-ui';
+import TooltipContent from '../../components/Calendar/subcomponents/TooltipContent.js';
+import styles from '../../utils/styles/CalendarStyles/Calendar.module.css';
+import { TimeTableCell, DayScaleCell } from '../../components/Calendar/Calendar.js';
+import { data } from '../../components/Calendar/data/dummyData.js';
+
+//import render Calendar component
+
+
+
+//use username from GET request, current is just mock data
+let username = 'Matt';
+
+
+export default function Calendar () {
+//import user state (mentor/mentee)
+
+  const [appointmentData, setAppointmentData] = useState([]);
+
+
+    useEffect(() => {
+      setAppointmentData(data);
```

Comment:
Later refactor to use SSG or SSR when fetching data. should mostly not need to use useEffect hook in /pages components

Reply:
yea we're just waiting for api calls, this was temporary